app-misc/qlcplus/qlcplus-4.12.1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils udev virtualx xdg
+
+DESCRIPTION="A software to control DMX or analog lighting systems"
+HOMEPAGE="https://www.qlcplus.org/"
+SRC_URI="https://github.com/mcallegari/${PN}/archive/QLC+_${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/qlcplus-QLC-_${PV}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	|| ( dev-embedded/libftdi:1 dev-embedded/libftdi:0 )
+	dev-libs/glib:2
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia:5[widgets]
+	dev-qt/qtnetwork:5
+	dev-qt/qtscript:5
+	dev-qt/qtwidgets:5
+	media-libs/alsa-lib
+	media-libs/libmad
+	media-libs/libsndfile
+	sci-libs/fftw:3.0
+	virtual/libusb:0
+	virtual/libusb:1
+	virtual/udev
+"
+DEPEND="${RDEPEND}
+	dev-qt/qttest:5"
+
+src_prepare() {
+	default
+	sed -e "s:/etc/udev/rules.d:${EROOT}lib/udev/rules.d:" -i \
+		plugins/hid/hid.pro \
+		plugins/udmx/src/src.pro \
+		plugins/dmxusb/src/src.pro \
+		plugins/spi/spi.pro \
+		plugins/peperoni/unix/unix.pro || die
+	sed -e "/UDEVRULESDIR/s:/etc/udev/rules.d:$(get_udevdir)/rules.d:" \
+		-i variables.pri || die
+
+	## Remove Werror-flag since there are some warnings with gcc-9.x
+	sed -e "s/QMAKE_CXXFLAGS += -Werror/#QMAKE_CXXFLAGS += -Werror/g" \
+		-i variables.pri || die
+}
+
+src_configure() {
+	eqmake5
+}
+
+src_install() {
+	emake DESTDIR="${D}" INSTALL_ROOT="${D}" install
```

Comment:
Are both DESTDIR and INSTALL_ROOT used?

Reply:
I've tried all permutations (neither defined, only DESTDIR defined, only INSTALL_ROOT defined or both defined).
Both defined or only INSTALL_ROOT defined have the same result. If INSTALL_ROOT is not given (no matter if DESTDIR is used), a sandbox violation is reported.
I'll remove the DESTDIR parameter in the next commits. (Waiting for your answer regarding the libftdi dependency)

Reply:
Done